assets/javascripts/discourse/widgets/discourse-reactions-actions.js

```diff
@@ -172,9 +173,11 @@ export default createWidget("discourse-reactions-actions", {
     this._validTouch = true;
 
     cancel(this._touchTimeout);
+
     if (this.capabilities.touch) {
-      const root = document.getElementsByTagName("html")[0];
-      root?.classList?.add("discourse-reactions-no-select");
+      document
```

Comment:
Changed to use `toggle` instead of `add` to ensure we only ever have that class once.